bindings/pyroot_experimental/cppyy/cppyy/python/cppyy/__init__.py

```diff
@@ -55,7 +55,15 @@
 major, minor = sys.version_info[0:2]
 py_version_str = '{}_{}'.format(major, minor)
 libcppyy_mod_name = 'libcppyy' + py_version_str
-importlib.import_module(libcppyy_mod_name)
+try:
+    importlib.import_module(libcppyy_mod_name)
+except ModuleNotFoundError as err:
+    if err.name.startswith('libcppyy'):
```

Comment:
I would not add this extra check here, if the import fails we error out and we provide a suggestion to the user about what can be going on.